### PR TITLE
pytester: ignore files used to obtain current user metadata

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
… fd leak detector

### CODE REVIEW

File: _pytest/pytester.py [hunk 1]
```diff
@@ -22,6 +22,11 @@
 from _pytest.assertion.rewrite import AssertionRewritingHook
 
 
+IGNORE_PAM = [  # filenames added when obtaining details about the current user
```
Comment (nicoddemus) [inline _pytest/pytester.py]: What's "PAM"?

File: changelog/2784.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+pytester: no ignore files used to obtain current user metadata in the fd leak detector
```
Comment (nicoddemus) [inline changelog/2784.bugfix]: IIUC this should be "ignore files", not "no ignore files", right?

Comment (nicoddemus) [inline changelog/2784.bugfix]: Nit: please add a "." at the end of the sentence. 😉

File: _pytest/pytester.py (hunk 1 above)
Comment (RonnyPfannschmidt) [reply _pytest/pytester.py]: thanks for noting, pam comes frim the plugablle authentification modules, the SSS module is the one that opens this file, constant however is indeed named bdly

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt gentle ping.

**Comment by RonnyPfannschmidt** [comment]
will retry after #3016

**Pull request merged**